core/src/main/java/io/varhttp/VarResponseHeader.java

```diff
@@ -45,6 +45,6 @@ public void redirect(URL url) {
 
 	@Override
 	public void addCookie(Cookie cookie) {
-		this.response.addCookie(cookie);
+		this.response.setHeader("Set-Cookie", cookie.getName() + "=" + cookie.getValue() + "; Path=" + cookie.getPath());
```

Comment:
Why not use the existing addcookie method? It also sets secure etc?

Reply:
The existing addCookie does not respect the path you set. So it always set /api/frontend-support for the cookie and thus other endpoins don't work, those are the errors we were seeing yesterday